src/message_interface/response.rs

```diff
@@ -44,6 +44,8 @@ pub enum Response {
     SentTransfer(TransferResult),
     /// TryCollectOutputs and CollectOutputs response.
     SentTransfers(Vec<TransferResult>),
+    /// IsStrongholdPasswordAvailable
```

Comment:
Comment is different than the variant ?

Reply:
The comment is the message type which returns this variant

Reply:
Sure but can't we harmonise ?

Reply:
Some other comments end with `response.`, should we rather use `Response for` before each of the message types?

Reply:
Yeah that sounds good, but I would also make the message type and the response more similar, it's weird that the `Is` is not at the same position

Reply:
Nah it's fine, let's keep it that way